include/bio/record.hpp

```diff
@@ -198,6 +204,7 @@ struct record : std::tuple<field_types...>
     }
     //!\}
 
+//!\brief Don't know how doxygen comform this will be. \todo check this.
```

Comment:
If we add the doxygen within the MACRO definition and tell doxygen to expand the macro as defined, it should be good.

Reply:
Doesn't work :(